modules/getting-started-web-console-login.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-web-console-login_{context}"]
+= Logging in to the web console
+
+You can log in to the {product-title} web console to access and manage your cluster.
+
+.Prerequisites
+
+* {product-title} web console login credentials.
+
+.Procedure
+
+. Log in to the {product-title} web console using your login credentials.
+
+You are redirected to the *Projects* page. For non-administrative users, the default view is the *Developer* perspective. For cluster administrators, the default view is the *Administrator* perspective.
```

Comment:
Should this mention that users that don't have admin perms won't see the Admin perspective?